lib/Sema/CSDiagnostics.cpp

```diff
@@ -6224,3 +6224,15 @@ bool CoercionAsForceCastFailure::diagnoseAsError() {
       .highlight(getSourceRange());
   return true;
 }
+
+bool KeyPathRootTypeMismatchFailure::diagnoseAsError() {
+  auto locator = getLocator();
+  assert(locator->isKeyPathRoot() && "Expect a key path root");
```

Comment:
Expect -> Expected

Reply:
I think "Expect" is fine for this 👍

Reply:
Infinitive here does not make sense semantically: "the assertion has been hit because the method *expected* the locator to be a key path root".

Reply:
Fixed!

Reply:
Thank you!